src/Form/index.tsx

```diff
@@ -254,6 +295,17 @@ function Form({
   // When the active step changes, recalculate the dimensions of the new step
   const stepCSS = useMemo(() => calculateStepCSS(activeStep), [activeStep]);
 
+  const getFormContextHelper = (newStep = activeStep) => {
+    return getFormContext(newStep, {
+      history,
+      client,
+      updateFieldOptions,
+      setUserProgress,
+      steps,
+      activeStep
```

Comment:
is activeStep even needed if we're using newStep already?

Reply:
oh, yeah think I can change the activeStep.key to newStep.key and clean up the code a bit
``` setFormCompletion: (flag: boolean) =>
          client.registerEvent({
            step_key: activeStep.key,
            event: 'load',
            completed: flag
          }),```